src/main/res/values/strings.xml

```diff
@@ -101,7 +101,7 @@
   <string name="over_wifi">Only on Wi-Fi</string>
   <string name="parsing_index_error_description">Could not parse the index file.</string>
   <string name="password">Password</string>
-  <string name="password_is_not_specified">Type in the password</string>
+  <string name="password_is_not_specified">Type in the password first</string>
```

Comment:
"Type in the password first" is not good here. Look at this:

<p>
<img src="https://user-images.githubusercontent.com/24494863/86565658-c8ca9100-bf57-11ea-98c1-250ced2b26e9.png" width="45%" />
<img src="https://user-images.githubusercontent.com/24494863/86565661-c9fbbe00-bf57-11ea-9a93-197a17700af9.png" width="45%" />
</p>

Both username and password are optional. But password is required when username specified, and vice versa. This is why I think "Password is not specified" should be like "Username is not specified".

Reply:
I mean, it's not an "iterative" process like you enter the username first and password then, like when you're trying to sign in somewhere. It's just two optional fields for the same purpose — authenticate in repository if necessary.

Reply:
```suggestion
  <string name="password_is_not_specified">Password missing</string>
```